website/blog/2019-06-26-7.5.0.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: post
+title:  "7.5.0 Released: dynamic import and F# pipelines"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2019-06-17 0:00:00
+categories: announcements
+share_text: "Babel 7.5.0 Released"
+---
+
+Today we are releasing Babel 7.5.0!
+
+This release includes improved support for a few ECMAScript proposals: the "F#" variant of the Stage 1 pipeline operator and an official plugin for the Stage 4 dynamic `import()` proposal (along with `preset-env` support).
```

Comment:
Note: we should be consistent when we reference the `"F#" variant` vs `F# variant` throughout

Reply:
I'll remove the quotes where they are used, since the proposals' READMEs don't use them.